cinema/tests/test_movie_api.py

```diff
@@ -157,3 +158,213 @@ def test_image_url_is_shown_on_movie_session_detail(self):
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
+
+
+class UnauthenticatedMovieApiTests(TestCase):
+    def setUp(self):
```

Comment:
add typehint both for finction parameters and return value for all tests functions